task1-gradient-descent/gradient_descent_soner.ipynb

```diff
@@ -0,0 +1,1627 @@
+{
```

Comment:
<p>there is good answer from @beyza </p><p> </p><p> </p><p> </p><ul><li>Normalization is really affected by outliers, because we use min and max values to shape our data. But standardization isn't affected by outliers that much, because we use mean and standart deviation. </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>